pkgs/desktops/gnome-3/3.24/core/gdm/src.nix

```diff
@@ -1,10 +1,10 @@
 # Autogenerated by maintainers/scripts/gnome.sh update
 
 fetchurl: {
-  name = "gdm-3.22.0";
+  name = "gdm-3.25.4.1";
```

Comment:
You probably should use GDM 3.24.2, odd minor versions are considered unstable in most GNOME apps.